### ISSUE TITLE
2 translate buggs

### ISSUE THREAD

**Issue by Molle500** [RocketChat/Rocket.Chat#2956]
I found 2 issues with your translate system. 

1. When you edit a message you have "esc to" and "cancel". It seems like there is only one cancel text to translate. In our language (Swedish) we use two different words for cancel and would be nice if you could add a separate translate for this cancel i have marked in the picture.

2. "users" under memberlist do not react to the translated text for "users" so this one should be linked to the "users" translate as well

tl;dr

Separate this cancel text to a own translated text. Link the user text to the translated text.

![translate](https://cloud.githubusercontent.com/assets/17712562/14653535/ac380a18-0679-11e6-857c-3255d0ce9f19.png)

**Comment by marceloschmidt** [comment]
We use the word "cancel" in so many places. Is this the only place where it needs to be a different word? 
As for the "users" word, I don't know what is it that you're requesting... this sentence comes from the `Showing_online_users` translation key.

**Comment by Molle500** [comment]
@marceloschmidt Yes it's the only place where it needs to be different :)

Ah yes nice find :) i was confused becouse the previous translator forgot to change "user" so the first part was in swedish and user was in english so i thought "users" were separate :). However the "users"  is fixed. If you could add the cancel i would really appreciate it!

// Mokke

**Comment by marceloschmidt** [comment]
@Molle500 I've added a new key for translating that specific 'cancel' word. The key is `Cancel_message_input`

**Comment by Molle500** [comment]
Great, thanks!
